imagery/i.ortho.photo/i.ortho.camera/i.ortho.camera.html

```diff
@@ -73,20 +73,20 @@ <h2>DESCRIPTION</h2>
 <p>
 Please provide the following information
 <pre>
-	Fid#	FID ID		  X          Y
-
-	1__	_____		0.0___	0.0___
-	2__	_____		0.0___	0.0___
-	3__	_____		0.0___	0.0___
-	4__	_____		0.0___	0.0___
-	5__	_____		0.0___	0.0___
-	6__	_____		0.0___	0.0___
-	7__	_____		0.0___	0.0___
-	8__	_____		0.0___	0.0___
-	9__	_____		0.0___	0.0___
-	10_	_____		0.0___	0.0___
-
-		     next:  end__
+    Fid#    FID ID          X          Y
+
+    1__    _____        0.0___    0.0___
+    2__    _____        0.0___    0.0___
+    3__    _____        0.0___    0.0___
+    4__    _____        0.0___    0.0___
+    5__    _____        0.0___    0.0___
+    6__    _____        0.0___    0.0___
+    7__    _____        0.0___    0.0___
+    8__    _____        0.0___    0.0___
+    9__    _____        0.0___    0.0___
+    10_    _____        0.0___    0.0___
+
+             next:  end__
```

Comment:
It's ok for the docs, but the tool really does output tabs?

Reply:
I don't see if it's tabs or spaces. Just noticing that if the tool was outputting tabs, now the docs would have spaces.

Reply:
It is spaces now and looks as before:

![image](https://github.com/user-attachments/assets/2660e4cc-d535-4041-b1b8-94c31679c80e)